ceph.md

```diff
@@ -0,0 +1,256 @@
+        store_description = "RBD backend"
+        rbd_store_pool = images
+        rbd_store_user = openstack
+```
+
+## Configure Cinder
+
+Use a `customServiceConfig` to pass overrides to Cinder's
+configuration file. For example, the sample
+[core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml)
+has the following in the `OpenStackControlPlane` CR:
+
+```
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+spec:
+  extraMounts:
+    ...
+  cinder:
+    template:
+      cinderVolumes:
+        ceph:
+          customServiceConfig: |
+            [DEFAULT]
+            enabled_backends=ceph
+            [ceph]
+            volume_backend_name=ceph
+            volume_driver=cinder.volume.drivers.rbd.RBDDriver
+            rbd_ceph_conf=/etc/ceph/ceph.conf
+            rbd_user=openstack
+            rbd_pool=volumes
+            rbd_flatten_volume_from_snapshot=False
+            rbd_secret_uuid=$FSID
+```
+The `$FSID` value above should contain the actual FSID as described
+in the "Create a Ceph Secret" section. The FSID itself does not need
+to be considered secret.
+
+## Configure Nova
+
+Use the `OpenStackDataPlane`
+[NovaTemplate](https://openstack-k8s-operators.github.io/dataplane-operator/openstack_dataplanerole/#novatemplate)
+to pass a `customServiceConfig`.
+
+```
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlane
+spec:
+<snip>
+  roles:
+    edpm-compute:
+    <snip>
+        nova:
+          cellName: cell1
+          customServiceConfig: |
+            [libvirt]
+            images_type=rbd
+            images_rbd_pool=vms
+            images_rbd_ceph_conf=/etc/ceph/ceph.conf
+            images_rbd_glance_store_name=default_backend
+            images_rbd_glance_copy_poll_interval=15
+            images_rbd_glance_copy_timeout=600
+            rbd_user=openstack
+            rbd_secret_uuid=$FSID
+          deploy: true
+          novaInstance: nova
+```
+The `$FSID` value above should contain the actual FSID as described
+in the "Create a Ceph Secret" section. The FSID itself does not need
+to be considered secret.
+
+## Full Examples
+
+The examples above are focussed on showing how a
+single `OpenStackControlPlane` and `OpenStackDataPlane`
+CR can be modified to include Ceph configuration by adding
+`extraMounts` and `customServiceConfig`. Links to complete
+examples are below.
+
+- `OpenStackControlPlane`: [core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml)
+- `OpenStackDataPlane`: [dataplane_v1beta1_openstackdataplane_ceph.yaml](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplane_ceph.yaml)
```

Comment:
This link will work after this PR merges

https://github.com/openstack-k8s-operators/dataplane-operator/pull/216

Reply:
ack